eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/SubmittedVariantAccessioningDatabaseService.java

```diff
@@ -54,4 +60,21 @@ public SubmittedVariantAccessioningDatabaseService(SubmittedVariantAccessioningR
         return new AccessionWrapper<>(entity.getAccession(), entity.getHashedMessage(), entity.getModel(),
                                       entity.getVersion());
     }
+
+    public AccessionWrapper<ISubmittedVariant, String, Long> getLastInactive(Long accession) {
+        return getLastInactive(accession, inactiveService);
+    }
+
+    static AccessionWrapper<ISubmittedVariant, String, Long> getLastInactive(
+            Long accession, InactiveAccessionService<ISubmittedVariant, Long, ?> inactiveService) {
+        IEvent<ISubmittedVariant, Long> lastEvent = inactiveService.getLastEvent(accession);
+        List<? extends IAccessionedObject<ISubmittedVariant, ?, Long>> inactiveObjects = lastEvent.getInactiveObjects();
+        if (inactiveObjects.isEmpty()) {
+            throw new IllegalArgumentException(
+                    "Accession " + accession + " is not inactive (not present in the operations collection");
+        }
+        IAccessionedObject<ISubmittedVariant, ?, Long> inactiveObject = inactiveObjects.get(inactiveObjects.size() - 1);
```

Comment:
We seem to be getting the last element of the returned list - Will the result be deterministic (same across multiple invocations)?

Reply:
after thinking about this, we already decided that if an accession is linked to several documents we would return only one of them, doesn't matter which. so here, returning one of the deprecated accessions, doesn't matter which document we return